src/cluecode/copyrights.py

```diff
@@ -4502,6 +4502,11 @@ def prepare_text_line(line):
 
         # normalize copyright signs, quotes and spacing around them
         .replace('"Copyright', '" Copyright')
+        
+        # normalize [C] and [c] to (c) before bracket removal
```

Comment:
this comment is not required, the comment above is explanatory and enough. Remove the extra spaces too